bot.py

```diff
@@ -1,3 +1,4 @@
+from click import pass_context
```

Comment:
What's the purpose of this library? Where is it used?

Reply:
That looks to be errata as a result of VSCode autoimporting something based on a typo. I'll remove it.

Reply:
Appreciated <3